README.md

```diff
@@ -51,66 +51,283 @@ directory to the web root of your server.
+   +   <subtitle>The most recent posts from {{ site[locale].name }}</subtitle>
+   ```
+
+### Disabling Internationalization for the Entire Site
+
+If you need to disable internationalization for the entire site, you can do so by:
+
+1. Remove all but the default locale from the locales block of [`src/_data/config.json`](src/_data/config.json):
+
+   ```diff
+   "locales": [
+   -   "en-CA",
+   -   "fr-CA"
+   +   "en-CA"
+   ],
+   ```
+
+2. Delete the `editor` block's `i18n` section in [`src/admin/config.yml`](src/admin/config.yml):
+
+   ```diff
+   editor:
+     preview: false
+   - i18n:
+   -   structure: multiple_folders
+   -   locales: ["en-CA", "fr-CA"]
+   -   default_locale: "en-CA"
+   ```
+
+3. Deleting all directories but the default locale in every directory within `/src/collections/`. For example, to
+   disable localization for pages, delete the `/src/collections/pages/fr-CA/` directory.
+4. Modify the [`src/admin/config.yml`](src/admin/config.yml) collection block's `i18n` and `folder` properties for all
+   collections:
+
+   ```diff
+   - i18n: true
+   - folder: "src/collections/<collection>/"
+   + folder: "src/collections/<collection>/en-CA/" # Look for content to edit in the default locale directory.
+   ```
+
+   Or, remove all subdirectories of each collection directory, moving the `en-CA` (or other default language)
+   directory's contents (with the exection of the `en-CA.11tydata.js` file) up into `src/collections/<collection>`. If
+   you do this, you'll also need to make the following change to the collection's directory data file,
+   `src/collections/<collection>.11tydata.js`:
+
+   ```diff
+   - const { EleventyI18nPlugin } = require("@11ty/eleventy");
+   - const i18n = require("eleventy-plugin-i18n-gettext");
+   const { generatePermalink } = require("eleventy-plugin-fluid");
+   
+   module.exports = {
+       layout: "layouts/post",
+       eleventyComputed: {
+   +       lang: data => data.defaultLanguage,
+   -       langDir: data => data.supportedLanguages[data.locale].dir,
+   +       langDir: data => data.supportedLanguages[data.defaultLanguage].dir,
+           permalink: data => {
+   -           const locale = data.locale;
+   -           return generatePermalink(data, "posts", i18n._(locale, "posts"));
+   +           return generatePermalink(data, "posts");
+           }
+       }
+   };
+   ```
+
+   If you choose this option you won't need to change the `folder` configuration in [`src/admin/config.yml`](src/admin/config.yml).
+5. Modify your Eleventy configuration file to disable string translation in `eleventy-plugin-fluid`:
+
+   ```diff
+   eleventyConfig.addPlugin(fluidPlugin, {
+   +  i18n: false
+      ...
+   });
+   ```
+
+   Then make sure you remove any uses of the [`localizeData`](https://github.com/fluid-project/eleventy-plugin-fluid#localizedata)
+   helper or [`eleventy-plugin-i18n-gettext` functions](https://github.com/sgissinger/eleventy-plugin-i18n-gettext#api)
```

Comment:
What's the proper way to handle the dates when localization is removed?

Reply:
The `formatDate` helper is still present in `eleventy-plugin-fluid`.